### DISCUSSION TITLE
Error While Initializeing shadcn in laravel project

### DISCUSSION THREAD

**Discussion by BilalSwl6** [shadcn-ui/ui#5620] [General]
i run `npx shadcn@latest init` but it's showing error
```✔ Preflight checks.
✔ Verifying framework. Found Laravel.
✔ Validating Tailwind CSS.
✖ Validating import alias.

No import alias found in your tsconfig.json file.
Visit https://ui.shadcn.com/docs/installation/laravel to learn how to set an import alias.

**Comment by ismaildasci** [comment]
Hey! 👋

The error you're seeing means `shadcn` is looking for an import alias in your `tsconfig.json` file, which Laravel projects typically don’t have by default. Here’s how to fix it:

1. **Create or Edit `tsconfig.json`**: In your project root, add a `tsconfig.json` file with the following content:

   ```json
   {
     "compilerOptions": {
       "baseUrl": ".",
       "paths": {
         "@/*": ["resources/js/*"]
       }
     },
     "include": ["resources/js/**/*"]
   }
   ```

   This setup creates an alias `@` that points to `resources/js`.

2. **Try Running the Command Again**: Now, try `npx shadcn@latest init` once more.

This should resolve the alias issue. Let me know if it works! 😊

**Reply by BilalSwl6** [reply]
i have still  same error, i am using laravel framework. what will be exact content for tsconfig.json?
